doc/cephfs/client-config-ref.rst

```diff
@@ -4,115 +4,115 @@
 
 ``client acl type``
 
-:Description: Set to ``"posix_acl"`` to enable POSIX ACLs.
+:Description: Set the ACL type. Currently, only possible value is ``"posix_acl"`` to enable POSIX ACL, or an empty string.
```

Comment:
Let's add a note here that this only takes effect when fuse_default_permissions is false